source/_integrations/airthings_ble.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: Airthings BLE
+description: Instructions on how to set up Airthings Devices over Bluetooth LE.
+ha_category:
+  - Environment
+  - Sensor
+ha_release: '2022.9'
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@vincegio'
+ha_domain: airthings_ble
+ha_bluetooth: true
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+Integrates Airthings BLE sensors into Home Assistant.
```

Comment:
Maybe you could tell a little more about what airthings is and a link to their website?